bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_fr.properties

```diff
@@ -257,5 +257,5 @@ offline.conf-error-not-supported-onecall-number-of-hours = Le paramètre 'foreca
 offline.conf-error-not-supported-onecall-number-of-days = Le paramètre 'forecastDays' doit être compris entre 0 et 7.
 
 # discovery result
-discovery.openweathermap.weather-and-forecast.api.local.label = Météo locale et prévisions
-discovery.openweathermap.uvindex.api.local.label = Indice UV local
+discovery.weather-and-forecast.local.label = Météo locale et prévisions
+discovery.air-pollution.local.label = Indice UV local
```

Comment:
Can you verify if this translation is correct? If not remove it from the French properties and let it be translated via Crowdin.

Reply:
Translation is correct.